### ISSUE TITLE
Keyword argument for loss function in torch_surrogate_training

### ISSUE THREAD

**Issue by tnies** [Computational-Biology-Aachen/MxlPy#37]
## Feature Request

**Is your feature request related to a problem/ another issue? Please describe.**
- [ ] Yes: 
- [X] No: Why do you consider this feature to be necessary? Depending on the problem and the network architecture an option for different loss function in `train_torch_surrogate`might be necessary

**Describe the the feature**
I would like to have a keyword argument with which I can change the loss function of the training process for instance `loss = torch.nn.MSE`or a custom loss function. If I missed it and its already implemented please close this issue

**Comment by tnies** [comment]
finished
